packages/ui-react/src/IdentityIcon/Polkadot.tsx

```diff
@@ -9,13 +9,17 @@ import Identicon from 'polkadot-identicon';
 
 type Props = BaseProps & {
   size: number,
-  value: string | Uint8Array
+  value: string | null
```

Comment:
At this level, I would still allow for both Uint8Array and string. (So basically keep at least the definition as-is)